### ISSUE TITLE
Can't start it

### ISSUE THREAD

**Issue by Radquu** [HerrEurobeat/steam-comment-service-bot#58]
I am typing the node start.js in the CMD and then it says this error

internal/modules/cjs/loader.js:1237update found. (online: 2.8.1 | local: 2.8.1)
    throw err;
    ^

SyntaxError: C:\Users\no\Desktop\steam-comment-service-bot-master\steam-comment-service-bot-master\logininfo.json: Unexpected string in JSON at position 160
    at parse (<anonymous>)
    at Object.Module._extensions..json (internal/modules/cjs/loader.js:1234:22)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (C:\Users\no\Desktop\steam-comment-service-bot-master\steam-comment-service-bot-master\src\controller.js:14:17)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)

**Comment by HerrEurobeat** [comment]
Hi, thanks for opening an issue.
It seems like you have a syntax error in your logininfo.json.
Please make sure your logininfo.json follows exactly this format:

```
{
	"bot0": ["username0", "password0"],
	"bot1": ["username1", "password1"],
	"bot2": ["username2", "password2"]
}
```
Make sure that after every line is a comma but not on the last line. (like in the example above)

**Comment by Radquu** [comment]
> Hi, thanks for opening an issue.
> It seems like you have a syntax error in your logininfo.json.
> Please make sure your logininfo.json follows exactly this format:
> 
> ```
> {
> 	"bot0": ["username0", "password0"],
> 	"bot1": ["username1", "password1"],
> 	"bot2": ["username2", "password2"]
> }
> ```
> 
> Make sure that after every line is a comma but not on the last line. (like in the example above)



There i putted the accounts. Is it incorrect to have the accounts here?

**Comment by HerrEurobeat** [comment]
What do you mean exactly? If you put your usernames and passwords inside the quotation marks in your logininfo.json you should be fine. 
But it is important that you get the Syntax exactly the same like in the example. And judging from the error you have made a mistake somewhere.

Please make sure that your usernames and passwords are inside the quotation marks, you haven't removed any bracket or comma and that after every line but the last line is a comma at the end.
Basically just follow the example at replace username and password with your credentials.

**Comment by Radquu** [comment]
> What do you mean exactly? If you put your usernames and passwords inside the quotation marks in your logininfo.json you should be fine.
> But it is important that you get the Syntax exactly the same like in the example. And judging from the error you have made a mistake somewhere in your logininfo.json.
> 
> Please make sure that your usernames and passwords are inside the quotation marks, you haven't removed any bracket or comma and that after every line but the last line is a comma at the end.
> Basically just follow the example at replace username and password with your credentials.

{
	"bot0": ["account 1", "pass 1"],
	"bot1": ["acc 2", "acc 2"],
	"bot2": ["acc 3", "acc 3"],
        "bot3": ["acc 4", "acc 4"],
        "bot4": ["acc 5", "acc 5"],
        "bot5": ["acc 6", "acc 6"],
        "bot6": ["acc 7", "acc 7"],
        "bot7": ["acc 8", "acc 8"]
}

isn't this good?

**Comment by HerrEurobeat** [comment]
Hm well looks fine. And you are still getting a `logininfo.json: Unexpected string in JSON at position 160` error?
Do your usernames/passwords contain weird characters? Maybe that breaks the Strings. 

My passwords only contain letters and numbers and I have never had a problem. There was one other person who had problems with too complicated passwords but I'm not sure anymore if it was the same error.

**Comment by Radquu** [comment]
> Hm well looks fine. And you are still getting a `logininfo.json: Unexpected string in JSON at position 160` error?
> Do your usernames/passwords contain weird characters? Maybe that breaks the Strings.
> 
> My passwords only contain letters and numbers and I have never had a problem. There was one other person who had problems with too complicated passwords but I'm not sure anymore if it was the same error.

It worked to start the node.js but i am a little stupid and i dont understand with the secret key.

**Comment by HerrEurobeat** [comment]
Glad to hear that it starts now. The secret key is only needed if you want to request comments via URL (for example if someone wants to incorporate my bot into their application).

If you are interested in that then you can find the secret key in the data.json (as it said in the console) and provide it as a query parameter like in the example found on localhost:3034/comment

**Comment by Radquu** [comment]
> Glad to hear that it starts now. The secret key is only needed if you want to request comments via URL (for example if someone wants to incorporate my bot into their application).
> 
> If you are interested in that then you can find the secret key in the data.json (as it said in the console) and provide it as a query parameter like in the example found on localhost:3034/comment

Thank you so much! But how can i find the Comment Bot. Sorry for asking too much but for me to understand it's like more hard.

**Comment by HerrEurobeat** [comment]
You have to add the account you provided as bot0 on Steam.

Since you know how your account is named you can just search for it on Steam. (Or since you had to log into them to create a community profile you should know their profile link)

**Comment by Radquu** [comment]
> You have to add the account you provided as bot0 on Steam.
> 
> Since you know how your account is named you can just search for it on Steam. (Or since you had to log into them to create a community profile you should know their profile link)

Oh, Thank you so much for your help! Good Luck with it.
